source/_integrations/xbox.markdown

```diff
@@ -18,6 +18,10 @@ The Xbox integration allows you to control Xbox One (or newer) consoles from Hom
 
 Home Assistant authenticates with Xbox Live through OAuth2 using the Home Assistant Cloud account linking service. Set up the integration through **Configuration -> Integrations -> Xbox**. Ensure you login using the Microsoft account that is linked to your Xbox consoles.
 
+<div class='note'>
+Because it uses the Home Assistant Cloud account linking service you **must** have either `cloud:` or `default_config:` in your `configuration.yaml`.
+</div>
```

Comment:
This ain't working @DubhAd, as it mixes Markdown & HTML.

```suggestion
<div class='note'>

Because it uses the Home Assistant Cloud account linking service you **must** have either `cloud:` or `default_config:` in your `configuration.yaml`.

</div>
```

For more information, please see the developer documentation.